### PR TITLE
DOCS-4464: add RUM org metrics 

### PR DESCRIPTION
**Pull request by gschatz-splunk**
add descriptions for org metrics 

https://signalfuse.atlassian.net/browse/DOCS-4464

### CODE REVIEW

File: signalfx-org-metrics/metrics.yaml [hunk 1]
```diff
@@ -1799,3 +1799,160 @@ sf.org.ui.num.pagevisits:
...
+  description: |
+    The number of calls to the `/v2/trace` endpoint.
+
+    * Dimension: orgId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numAddSpansCalls
+
+sf.org.rum.numAddSpansCallsByToken:
+ brief:  The number of calls to the ... endpoint for a specific access token.
```
Comment (gschatz-splunk) [inline signalfx-org-metrics/metrics.yaml:1837]: hi! @atoulme :) which endpoint is correct ? thanks!

Comment (atoulme) [reply signalfx-org-metrics/metrics.yaml:1837]: I'm sorry, I don't know this. A member of the RUM team can maybe help you here.

Comment (gschatz-splunk) [reply signalfx-org-metrics/metrics.yaml:1837]: got it, thanks!

**Comment by gschatz-splunk** [comment]
hi! @atoulme I confirmed with Seerut that all of the descriptions are correct. Could you please approve this? thanks! :)

File: signalfx-org-metrics/metrics.yaml [hunk 2]
```diff
@@ -1799,3 +1799,167 @@ sf.org.ui.num.pagevisits:
...
+  brief: The possibly compressed wire size of the payloads before the payloads are decompressed and decoded.
+  description: |
+    The possibly compressed wire size of the payloads before the payloads are decompressed and decoded.
+    
+    * Dimensions: orgId
+    * Data resolution: 10 seconds
+
+  metric_type: Counter
+  title: sf.org.rum.grossContentBytesReceived
+
```
Comment (atoulme) [inline signalfx-org-metrics/metrics.yaml:1953]: Can we please remove those empty lines?

File: signalfx-org-metrics/metrics.yaml [hunk 3]
```diff
@@ -1799,3 +1799,167 @@ sf.org.ui.num.pagevisits:
...
+    The bytes of all the spans accepted and counted by the metric numSpansReceived by token. 
+
+    * Dimensions: orgId, tokenId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpanBytesReceivedByToken
+
+
+sf.org.rum.grossSpanBytesReceivedByToken:
+    brief: The bytes (uncompressed) of all the spans received and counted by the metric grossSpansReceived before throttling or filtering by token.
```
Comment (atoulme) [inline signalfx-org-metrics/metrics.yaml:1913]: Why does it have a different data resolution from grossSpansReceived, out of curiosity?

File: signalfx-org-metrics/metrics.yaml (hunk 2 above)
Comment (gschatz-splunk) [reply signalfx-org-metrics/metrics.yaml:1953]: Yes, I'll fix that thanks!

File: signalfx-org-metrics/metrics.yaml (hunk 3 above)
Comment (gschatz-splunk) [reply signalfx-org-metrics/metrics.yaml:1913]: hi!  I would ask Matt Pound :)

Comment (atoulme) [reply signalfx-org-metrics/metrics.yaml:1913]: @mdubbyap hey Matt, would you please review those metrics and approve the PR if all looks good? Thanks!

**Comment by uagarwal-splunk** [comment]
Looks good

**Comment by gschatz-splunk** [comment]
hi! @atoulme , ready for you ! approved by Umang :) thanks for your patience and assistance Antoine!

**Review by atoulme — APPROVED** [review]

**Comment by gschatz-splunk** [comment]
thanks so much!

**Pull request merged**